pipeline/tests/powdr_std.rs

```diff
@@ -131,8 +131,7 @@ fn arith_large_test() {
     let f = "std/arith_large_test.asm";
     let pipeline: Pipeline<GoldilocksField> = make_simple_prepared_pipeline(f);
     test_mock_backend(pipeline);
-    // TODO We can't use P3 yet for this test because of degree 4 constraints.
-    //test_plonky3_with_backend_variant::<BabyBearField>(f, vec![], BackendVariant::Monolithic);
+    test_plonky3_with_backend_variant::<GoldilocksField>(f, vec![], BackendVariant::Monolithic);
```

Comment:
these can be collapsed into `regular_test_gl(f, vec![]);`

Reply:
unless we don't want to test pilcom which would be fair enough

Reply:
Nice! Done.